packages/web-components/fast-foundation/docs/guide/styling-components.doc.md

```diff
@@ -3,4 +3,91 @@ id: styling-components
 title: Styling Components
 sidebar_label: Styling Components
 custom_edit_url: https://github.com/microsoft/fast-dna/edit/master/packages/web-components/fast-foundation/docs/guide/styling-components.doc.md
----
\ No newline at end of file
+---
+## Applying CSS
+`@microsoft/fast-components` leverages `@microsoft/fast-element` to style components. For details on how to apply CSS, see [leveraging CSS](fast-element/leveraging-css.md).
+
+## Design System
+In FAST, the *design system* is a collection of properties and values that inform the visual design language of the components. These properties are managed and provided through implementations of the *DesignSystemProvider*. See the [*DesignSystemProvider* documentation](fast-foundation/fast-design-system-provider.md) for more information.
```

Comment:
I think this could be made more concise by making it ... "implementation of the [*DesignSystemProvider*](fast-foundation/fast-design-system-provider.md)."  Thus just making it a link to the documentation instead of stating that part.